### PR TITLE
recommend consistent use of one block splitter

### LINKED ISSUE #608: Cell marker causes pycodestyle error
I was wondering why [`#%%` is recommended](https://sphinx-gallery.github.io/stable/tutorials/plot_parse.html#sphx-glr-tutorials-plot-parse-py), because stickler-CI complains that it doesn't satisfy pycodestyle because there's no space after the comment marker. I hate to be pedantic (["... hobgoblin of little minds."](https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds)) but we depend on the CIs to review PRs so if there's an exception it's more challenging to review, sorry.

I'm really sorry if this issue is redundant. I looked through the existing issues searching for "cell" & "space" but couldn't find a relevant issue. I am just getting started with Sphinx gallery, and it's really awesome, thanks so much.

 Thanks!!!

### PR DESCRIPTION
**Pull request by mikofski**
* closes #608

### CODE REVIEW

**Comment by larsoner** [comment]
Failures are real

https://travis-ci.org/sphinx-gallery/sphinx-gallery/jobs/650199958#L939

File: doc/syntax.rst [hunk 1]
```diff
@@ -45,8 +45,9 @@ Jupyter Notebook for each example that is built).
 
 You can embed rST in your Python examples by including a line of  >= 20 ``#``
 symbols or ``#%%``. For compatibility reasons, ``# %%`` (with a space) can also
-be used but ``#%%`` is recommended for consistency. If using ``#``'s, we
-recommend using 79 ``#``'s, like this::
+be used but it is recommended to consistently use either ``#%%`` or ``# %%`` for
+all examples in the gallery. If using ``#``'s, we recommend using 79 ``#``'s,
```
Comment (lucyleeow) [inline doc/syntax.rst]: I would suggest:
```
You can embed rST in your Python examples by including a line of  >= 20 ``#`` 
symbols,  ``#%%`` or ``# %%``. For consistency, it is recommended that you use
only one of the above three options in your project.
```

**Comment by lucyleeow** [comment]
The test failure is because one of the tests checks that `plot_parse.py` is split correctly into text and code blocks. Since there haven't been many changes, the easiest thing to do would be to add the new changes to [`reference_parse.txt`](https://github.com/sphinx-gallery/sphinx-gallery/blob/master/sphinx_gallery/tests/reference_parse.txt) which is a pretty printed list of tuples where the 2nd element in each tuple is the content of the code or text block.

**Comment by mikofski** [comment]
[`test_gen_rst/test_bug_cases_of_notebook_syntax`](https://travis-ci.org/sphinx-gallery/sphinx-gallery/jobs/650627210#L850) is passing, but now [`test_full/test_embedded_links_and_styles`](https://travis-ci.org/sphinx-gallery/sphinx-gallery/jobs/650627210#L822) fails, and I can't figure out how they're related.


Here's the [traceback](https://travis-ci.org/sphinx-gallery/sphinx-gallery/jobs/650627210#L938-L943):
<details>

```
_________________________ test_embed_links_and_styles __________________________
sphinx_gallery/tests/test_full.py:227: in test_embed_links_and_styles
    assert '.html#numpy.random.RandomState" title="numpy.random.RandomState" class="sphx-glr-backref-module-numpy-random sphx-glr-backref-type-py-class"><span class="n">np</span>' in lines  # noqa: E501
E   assert '.html#numpy.random.RandomState" title="numpy.random.RandomState" class="sphx-glr-backref-module-numpy-random sphx-glr-backref-type-py-class"><span class="n">np</span>' in '\n<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"\n  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitio...ot_numpy_matplotlib.rst.txt"\n          rel="nofollow">Page source</a>\n    </div>\n\n    \n\n    \n  </body>\n</html>'
```
</details

**Comment by larsoner** [comment]
That failure is unrelated and should be fixed in `master` (or maybe we broke it there), in any case if it's not fixed we can fix it elsewhere.

**Comment by larsoner** [comment]
Thanks @mikofski !

**Pull request merged**